recipes/libpsl/all/conanfile.py

```diff
@@ -0,0 +1,105 @@
+from conans import ConanFile, Meson, tools
+import os
+
+
```

Comment:
```suggestion
required_conan_version = ">=1.29.1"

```

`tools.remove_files_by_mask: https://github.com/conan-io/conan/releases/tag/1.29.1

Reply:
By the way, why do we need this if it is advised to use the latest Conan client version, especially if there was an explicit build error instead of "soft" error?

Reply:
It's adviced, but a lot of people are not aware of it. An meaningful error is better than a python call stack.